cypress/e2e/tests/migration/dynamic-report/issues/allIssues/sort.test.ts

```diff
@@ -36,27 +36,12 @@ describe(["@tier2"], "Issues sort validations", function () {
         application.analyze();
         cy.wait(2 * SEC);
         application.verifyAnalysisStatus("Completed");
-        Issues.openList();
-        validateSortBy("Issue");
     });
 
-    it("Sort issues by category", function () {
-        Issues.openList();
-        validateSortBy("Category");
-    });
-
-    it("Sort issues by effort", function () {
-        Issues.openList();
-        validateSortBy("Effort");
-    });
-
-    it("Sort issues by affected applications", function () {
-        Issues.openList();
-        validateSortBy("Affected applications");
-    });
-
-    after("Perform test data clean up", function () {
-        Analysis.open(true);
-        application.delete();
```

Comment:
Why has this line been removed ? The application is not being deleted as part of the clean up .

Reply:
Thanks for noticing this, it was done by mistake. Returned it back already.